pages/common/less.md

```diff
@@ -22,6 +22,10 @@
 
 `?{{something}}`
 
+- Tail currently opened file:
+
+`<Shift> F`
```

Comment:
Just `F` is sufficient. Above in the page it's mentioned `G` and `g`.

Reply:
Done